internal/dcache/replication_manager/models.go

```diff
@@ -215,6 +215,80 @@ func (req *WriteMvRequest) isValid() error {
 type WriteMvResponse struct {
 }
 
+type WriteMvExRequest struct {
```

Comment:
There's no need to define a new request type. WriteMVEx() does the same thing (writing to an MV) but does it differently, so it should use the same request type WriteMvRequest and returns the same response type WriteMvResponse.